project-apps-dev/ProgrammersOnly/env_files/mongo.env

```diff
@@ -0,0 +1,2 @@
+MONGO_INITDB_ROOT_USERNAME=root
```

Comment:
It doesn't solve the problem of storing username and password in repository

As a workaround I suggest uploading this file to s3 manualy. Nevertheless it should be automated somehow.